packages/gap-web-ui/src/components/summary-list/SummaryList.module.scss

```diff
@@ -3,3 +3,11 @@
 .gap-summary-list--key-weight-regular {
   font-weight: $govuk-font-weight-regular;
 }
+
+@media (min-width: 40.0625em) {
+  .govuk-summary-list.key-width-50percent-sm {
+      .govuk-summary-list__key {
+        width: 50%;
+      }
+  }
+}
```

Comment:
This something we want to apply to every summary list?

Reply:
The key column in the figma is significantly wider than the component default (30% vs 50%) I don't want to edit existing implementations